test/parallel/test-cluster-uncaught-exception.js

```diff
@@ -4,26 +4,20 @@
 // https://github.com/joyent/node/issues/2556
 
 const common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var fork = require('child_process').fork;
+const assert = require('assert');
+const cluster = require('cluster');
+const fork = require('child_process').fork;
 
-var MAGIC_EXIT_CODE = 42;
+const MAGIC_EXIT_CODE = 42;
 
-var isTestRunner = process.argv[2] != 'child';
+const isTestRunner = process.argv[2] !== 'child';
 
 if (isTestRunner) {
   var master = fork(__filename, ['child']);
-  master.on('exit', common.mustCall(function(code) {
-    assert.strictEqual(code, MAGIC_EXIT_CODE);
-  }));
-} else if (cluster.isMaster) {
-  process.on('uncaughtException', function() {
-    process.nextTick(function() {
-      process.exit(MAGIC_EXIT_CODE);
-    });
-  });
+  master.on('exit', common.mustCall(code => assert.strictEqual(code, MAGIC_EXIT_CODE)));
```

Comment:
Are you sure you ran `make -j8 test` (or just `make test`, either is fine) on these changes? I would expect the linter to complain about this line as being too long.